receiver/dockerstatsreceiver/testdata/mock/single_container/container.json

```diff
@@ -0,0 +1,218 @@
+{
+  "Id": "10b703fb312b25e8368ab5a3bce3a1610d1cee5d71a94920f1a7adbc5b0cb326",
```

Comment:
nit: can the json files be sorted for potentially easier maintenance/review over time?

Reply:
Done!